awx/main/migrations/0188_inbound_hop_nodes.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 4.2.6 on 2023-11-02 18:07
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('main', '0187_hop_nodes'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ReceptorAddress',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('address', models.CharField(max_length=255)),
+                ('port', models.IntegerField(default=27199)),
+                ('protocol', models.CharField(default='tcp', max_length=10)),
+                ('websocket_path', models.CharField(blank=True, default='', max_length=255)),
+                ('is_internal', models.BooleanField(default=False)),
+                ('peers_from_control_nodes', models.BooleanField(default=False)),
+            ],
+        ),
+        migrations.RemoveConstraint(
+            model_name='instancelink',
+            name='source_and_target_can_not_be_equal',
+        ),
+        migrations.AlterUniqueTogether(
+            name='instancelink',
+            unique_together=set(),
+        ),
+        migrations.RemoveField(
```

Comment:
Before this data is removed, I take it that a data migration needs to be added to create ReceptorAddress objects for old instances. It looks like that will also require re-ordering these steps.

Reply:
yep this is this planned work item we still need to implement